### PR TITLE
Notes on impact analysis and budgeting

### PR DESCRIPTION
**Pull request by drphilmarshall**
@DukeofSuffolk @leannep : I took a pass through the RMP (Matt's ticket branch) and added minimal text to support the cost and schedule impact analysis and subsequent risk-aware budgeting. I think this is ready for review, and you can ask me to take a second pass if that would be helpful.

I fixed a few other things as I went, as well. Feel free to correct these as needed. The main one is that I noticed that the document has a couple of sections which just define terminology, so I modified the section heading to make that clear. The actual management processes we have defined all seem to be in the "Identification and Assessment" section.

### CODE REVIEW

File: Identification-and-Assessment.tex [hunk 1]
```diff
@@ -30,49 +30,88 @@ \subsection{Create Risk Item}
...
+	\item Realized Risk Plan --- Description of event trigger, and the plan for how the risk will be addressed once it is realized.
 	\item All fields under Analyze Risk Quantitative --- note that these do not automatically populate any required fields.
 	\item Existential Risk
 \end{itemize}
 
 Optional fields:
 \begin{itemize}
 	\item Overall Impact --- Impact category that can override the automatically generated cost and schedule impact categories (sets to most severe)
-	\item Cost or Schedule Impact Description
+	% \item Cost or Schedule Impact Description. PJM: See above, I think we always need a description any time we specify an impact.
```
Comment (leannep) [inline Identification-and-Assessment.tex]: Agree   -- in the final merge you can remove the '%' comment line

File: Identification-and-Assessment.tex [hunk 2]
```diff
@@ -30,49 +30,88 @@ \subsection{Create Risk Item}
...
-	\item Cost Impact --- According to risk table (need to add risk table to RDO-71)
+	\item Cost Impact --- According to risk table (need to add risk table to RDO-71).
 	\item Likelihood --- According to risk table (need to add risk table to RDO-71)
 	\item Schedule Impact --- According to risk table (need to add risk table to RDO-71)
-	\item Cost or Schedule Impact Description, if over certain severity.
-	\item Cost or Schedule fields under Analyze Risk Quantitative, if over certain severity.
-	\item Plan Type, and a minimum of one related Reponse Plan
-	\item Realized Risk Plan --- Description of event trigger
+	\item Cost and/or Schedule Impact Description
+	% if over certain severity. PJM: I think we always want a brief description of any impact we specify.
```
Comment (leannep) [inline Identification-and-Assessment.tex]: Agree   -- in the final merge you can remove the '%' comment line

File: Identification-and-Assessment.tex [hunk 3]
```diff
@@ -30,49 +30,88 @@ \subsection{Create Risk Item}
...
 	\item Related Actions
 \end{itemize}
 
 Verify assessment in the automatically generated fields under Analyze Risk Score.
 
+When analyzing a risk that is the parent of $N$ child risks:
+\begin{itemize}
+	\item First analyze the children separately, i.e. estimating Residual Probability $P_k$, Likely Cost $C_k$, and Likely Delay $S_k$ for the $k$-th risk, and repeating for all $k$ in the range $(1 \ldots N)$.
+	\item For each risk, calculate the Cost Exposure $E_k = C_k \times P_k$.
+	\item The combined Cost Exposure for the parent risk is just the sum of the Cost Exposures of its children, $E_{\rm total} = \sum_{k=1}^N E_k$.
```
Comment (leannep) [inline Identification-and-Assessment.tex:74]: Is it really just a linear sum? This assumes that all children are non-overlapping and fully independent.  I would not rule out some exposure/cost overlap of the children.  The combined cost exposure of the parent could be less. If if is equal, that implies that the flowdown to children is complete, which I'm not sure is the case either.  Do all the child risks capture all the risk in the parent? I'm not sure they do

File: Identification-and-Assessment.tex [hunk 4]
```diff
@@ -30,49 +30,88 @@ \subsection{Create Risk Item}
...
+	\item The Likely Cost of the parent risk is the sum of the Likely Costs of its children, $C_{\rm total} = \sum_{k=1}^N C_k$.
```
Comment (leannep) [inline Identification-and-Assessment.tex:75]: Same argument as for Cost Exposure

File: Identification-and-Assessment.tex [hunk 5]
```diff
@@ -30,49 +30,88 @@ \subsection{Create Risk Item}
...
+	\item The Residual Probability of the parent risk is its Cost Exposure divided by its Likely Cost, $P_{\rm total} = E_{\rm total} / C_{\rm total}$.
+	This is just the cost-weighted average of the child risks' Residual Probabilities, and is the most meaningful definition of probability for parent risks.
+	\item Other quantitative attributes of parent risks can be computed by simple summation (e.g. Minimum Cost, Maximum Delay, Schedule Exposure, etc).
+	\item Pre-response quantities of parent risks, such as Impact Scores, can be reverse engineered from their Residual Probabilities.
+\end{itemize}
```
Comment (leannep) [inline Identification-and-Assessment.tex:80]: Same argument for most of these child risks as for Cost Exposure. Perhaps we could say something like ... "To first order, the estimated Cost Exposure of the parent risk is the sum of the Cost Exposure of its children". Ditto for all other parameters

File: Identification-and-Assessment.tex [hunk 6]
```diff
@@ -30,49 +30,88 @@ \subsection{Create Risk Item}
...
+\subsection{Addressing Realized Risks}
+
+Triggering a Realized Risk:
+Review of all Response Plans and Actions by Risk Owner, and follow-up review with department management.
+Risk Owner proposes change of Status from Active to Realized.
+Review with RROB is required if severity is above threshold (determined by RROB Chair).
+
+Addressing a Realized Risk:
+Risk Owner works with department management to submit a Request Beyond Target (RBT) for additional resources needed to address the risk (see RTN-005 for the RBT process).
+RBT is reviewed by RDO, which works with department for any needed escalation to NOIRLab, SLAC or the funding agencies or Resource Forum.
```
Comment (leannep) [inline Identification-and-Assessment.tex:100]: What is "Resource Forum" ?

**Review by leannep — CHANGES_REQUESTED** [review]

File: Identification-and-Assessment.tex (hunk 6 above)
Comment (leannep) [reply Identification-and-Assessment.tex:100]: Follow up from slack discussion.  Add a sentence making it clear that if a parent risk is broken down into child risks, then child risks must be non-overlapping, fully independent and fully cover the parent risk. This way the assessment  in terms of Cost Exposure, etc, is just a simple linear sum of the child risks

Comment (drphilmarshall) [reply Identification-and-Assessment.tex:100]: Resource Forum = US agencies plus international in-kind contribution program's funding agencies. So its a bit redundant with "funding agencies" but not incorrect

File: Identification-and-Assessment.tex (hunk 3 above)
Comment (drphilmarshall) [reply Identification-and-Assessment.tex:74]: I committed a note that addresses this: essentially we require that child risks be independent and non-overlapping and a complete set in order for the cost exposures (and other quantities) to be simple/linear sums. This is the guidance I gave Chuck when he split T-32 into T-301 and T-302.

File: Identification-and-Assessment.tex (hunk 2 above)
Comment (drphilmarshall) [reply Identification-and-Assessment.tex]:
```suggestion
```

File: Identification-and-Assessment.tex (hunk 1 above)
Comment (drphilmarshall) [reply Identification-and-Assessment.tex]:
```suggestion
```

**Review by drphilmarshall — COMMENTED** [review]
Using the PR suggest functionality to make some minor edits, removing latex comments for the final version as requested by @leannep

**Comment by drphilmarshall** [comment]
@DukeofSuffolk I believe you can merge this whenever you are ready, I addressed @leannep 's comments.

**Pull request merged**